.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = crlf
+indent_size = 4
+indent_style = space
```

Comment:
this is the biggest debate. I thought we were using tab size: 4 :(

Reply:
fixed :)